packages/amplication-server/src/core/auth/auth.service.ts

```diff
@@ -386,7 +363,7 @@ export class AuthService {
         previewAccountType,
         previewAccountEmail,
       },
-      identityProvider: `${IDENTITY_PROVIDER_PREVIEW_ACCOUNT}_${previewAccountType}`,
+      identityProvider,
```

Comment:
I think we still need to add the preview account type, although we have only one now, but refactoring this only when we have another one will change the event name (if I'm not mistaken)

Reply:
I agree, this is done here at line 356: https://github.com/amplication/amplication/pull/7916/files/25550b92b9c582f87967b2ff0b2f6982414a5387#diff-368a0067447ce7f9a858f275793696cb4a7fc7111e2c14caba62cfb5febf3eddR356